qtglean/pings.yaml

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This file defines the pings that are recorded by the Glean SDK.
+# Their code APIs is automatically generated, at build time using,
+# the `glean_parser` PyPI package.
+
+---
+$schema: moz://mozilla.org/schemas/glean/pings/2-0-0
+
+vpnsession:
+  description: |
+    Data for one VPN session (start/end time, session-based metrics, etc.)
+  include_client_id: true
+  send_if_empty: true
+  bugs:
+    - https://mozilla-hub.atlassian.net/browse/VPN-4095
+  data_reviews:
+    - N/A, will be done for metrics within
+  notification_emails:
+    - mcleinman@mozilla.com
```

Comment:
Also include vpn@mozilla.com here, just in case.

Reply:
Added, thanks.